SparkPost/2web2ui
# SA-2027: Fix parsing 20+ metrics

### What Changed

- Added option to qs.parse so it would always parse to array

### How To Test

- go to signals/analytics.
- Select 20+ metrics
- Verify that everything still works correctly
### To Do
